.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -52,7 +52,7 @@ concurrency:
 
 env:
   # renovate: datasource=github-releases depName=cilium/cilium-cli
-  cilium_cli_version: v0.15.14
+  cilium_cli_version: v0.15.19
```

Comment:
@pchaigno: This workflow calls `conn-disrupt-test` with `extra-connectivity-test-flags: --test '!no-missed-tail-calls'`. After the cilium-cli bump, should this be renamed into `... '!no-unexpected-packet-drops'`, because of https://github.com/cilium/cilium-cli/commit/4880c91a726ddbad3363cc48f4bb8c11ed1150bd? (I can add it to my own PR if this is the case.)

Reply:
Ouch, yes, good catch!

It may not be the only place where I missed that change 😬😬😬

Reply:
This is the only occurrence I found on this branch.

Fix in https://github.com/cilium/cilium/pull/30010/commits/95a2cf9a49cc3e6bfb2b6102f655b6ba495f71be

Reply:
There are some other occurrences in a different workflow on `main` and `1.15`, I opened another PR: https://github.com/cilium/cilium/pull/30211